src/spells.cpp

```diff
@@ -1137,9 +1137,9 @@ bool RuneSpell::executeUse(Player* player, Item* item, const Position&, Thing* t
 
 	if (needTarget) {
 		if (!target) {
-			Tile* toTile = g_game.map.getTile(toPosition);
+			auto toTile = g_game.map.getTile(toPosition);
 			if (toTile) {
-				const Creature* visibleCreature = toTile->getBottomVisibleCreature(player);
+				auto visibleCreature = toTile->getBottomVisibleCreature(player);
```

Comment:
Could this be `const auto`
```suggestion
				const auto visibleCreature = toTile->getBottomVisibleCreature(player);
```